modules/sugar/src/main/ts/ephox/sugar/api/properties/Css.ts

```diff
@@ -1,12 +1,14 @@
-import { console, HTMLElement, window, Element as DomElement, Node as DomNode } from '@ephox/dom-globals';
+import { console, CSSStyleDeclaration, Element as DomElement, HTMLElement, Node as DomNode, window } from '@ephox/dom-globals';
 import { Arr, Obj, Option, Strings, Type } from '@ephox/katamari';
 import * as Style from '../../impl/Style';
 import * as Body from '../node/Body';
 import Element from '../node/Element';
 import * as Node from '../node/Node';
 import * as Attr from './Attr';
 
-const internalSet = function (dom: DomNode, property: string, value: string) {
+const styleKey = 'style';
```

Comment:
I'm curious why you pulled this out?

Reply:
It was a duplicate string, so it helps to split it out to reduce the size of the code.

Reply:
So I tested this more and it turns out the minification is actually putting the string back inside the actual function calls. So this ended up being meaningless anyways, so I'll revert it.

Also goes to show that minification tools don't always make the right decision 🤷‍♀️

Reply:
```
'style''style'
var x='style'xx
```
It is shorter in this case 😉

Reply:
Oh, I missed rollup had pulled out the `preserve` function as it's not used in most cases. I should of looked harder, as I was working on the basis of 5 strings, eg:
```
'style''style''style''style''style'
var x='style'xx
```

Reply:
Either way, it still shows the original change wasn't needed 😄

Reply:
oh, there were 5, sorry I mis-counted. I have a similar problem in BuckleScript, actually, it will constant inline variables for performance when sometimes I'd much rather it keep them as variables for code size